spec/linter-stylelint-spec.js

```diff
@@ -7,117 +7,136 @@ describe('The stylelint provider for Linter', () => {
 
   beforeEach(() => {
     atom.workspace.destroyActivePaneItem();
+    atom.config.set('linter-stylelint.usePreset', true);
+
     waitsForPromise(() => {
-      atom.packages.activatePackage('linter-stylelint');
```

Comment:
I think that should work, haven't tried simplifying that in a while because we were just happy we got _something_ working. I'll try that in a sec over in `linter-php`.